e2e/src/app.e2e-spec.ts

```diff
@@ -64,6 +64,8 @@ describe('Test App', () => {
             await browser.wait(EC.presenceOf(element(by.css('.rm-value'))), timeout,
                 'Wait for read value to be visible.');
 
```

Comment:
please add a comment that this is necessary because the DOM element has been re-created

Reply:
added in 4768a6b